docs/cadence-to-temporal.md

```diff
@@ -0,0 +1,66 @@
+---
+id: cadence-to-temporal
+title: Cadence to Temporal migration highlights
+sidebar_label: Migrate from Cadence
+---
+
+This page highlights the key changes between Cadence and Temporal that you will need to account for when migrating.
+
+There is no direct upgrade path from Cadence to Temporal.
+The Temporal Server requires a fresh install, and we recommend choosing one of the [latest releases](https://github.com/temporalio/temporal/releases).
+
+The [changelog blog post](/blog/temporal-v0.28.0-changelog) includes many of the core changes that occurred prior to Temporal's v1.0.0 release.
+
+## Key differences
+
+1. **Cadence uses Thrift/TChannel | Temporal uses [Protobuf](https://developers.google.com/protocol-buffers)/[gRPC](https://grpc.io/)**
+
+When using Temporal, this means the following things:
+
+  - Communication over can be encrypted via [TLS](/docs/server-configuration/#tls).
+  - Custom service errors backed by standard [gRPC error codes](https://pkg.go.dev/google.golang.org/grpc/codes).
+  - Errors are serialized using the Protobuf type `Failure` and they be chained together and passed across different SDKs in different languages.  
+  - All payloads (Workflow input, Activity input, etc...) sent to Temporal now have headers and data fields.
+  - Standard [gRPC health service](https://github.com/grpc/grpc/blob/master/doc/health-checking.md) is used for health checks.
+  - Standard Protobuf types are used for [time](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#timestamp) and [duration](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration).
+
```

Comment:
Another thing worth to mention here, I believe that persistence layer is also using protobufs blobs for most of the data for both MySQL and Cassandra. In Cadence Thrift was used in MySQL only.